modules/light-clients/07-tendermint/types/misbehaviour_handle.go

```diff
@@ -68,6 +94,7 @@ func (cs ClientState) CheckMisbehaviourAndUpdateState(
 	}
 
 	cs.FrozenHeight = tmMisbehaviour.GetHeight().(clienttypes.Height)
```

Comment:
The frozen height is the height of the offending header. In the case of time misbehaviour, it is the existence of the larger height that breaks monotonicity.

This feeds into the intuition we had with submitting misbehaviour that lowers frozen height. That logic rested on the understanding that frozen height was height of offending header. You could lower frozen height by submitting proof of an earlier offending header.

No longer matters now, but if we choose to re-enable this would follow that intuition.

Reply:
Couldn't an attacker construct a fill in header causing the larger height to become the offending header despite it being signed by > 66% of validators and thus steal funds since the frozen height would have been at the larger height